<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConflictEditor</class>
 <widget class="QWidget" name="ConflictEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conflicts</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="toolTip">
        <string>List of all conflicted files.</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="iconSize">
        <size>
         <width>43</width>
         <height>43</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditLocal">
         <property name="toolTip">
          <string>Local version.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonKeepLocal">
         <property name="toolTip">
          <string>Keep local version and overwrite remote version.</string>
         </property>
         <property name="text">
          <string>keep &amp;local</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditRemote">
         <property name="toolTip">
          <string>Remote version.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonKeepRemote">
         <property name="toolTip">
          <string>Keep remote version and overwrite local version.</string>
         </property>
         <property name="text">
          <string>keep &amp;remote</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
